src/main/resources/EIBackendInstancesInformation.json

```diff
@@ -1 +1 @@
-[]
+[]
```

Comment:
What changed here?

Reply:
Nothing. I accidentally committed a modified file and then blanked it. For some reason it's part of my request now...